src/components/Proposal/View/ProposalVoting/GetInvolvedQuestion.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 
 import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
 import useFormatMessage, { useIntl } from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { kebabCase } from 'lodash'
```

Comment:
Please, change the import to:

```suggestion
import kebabCase from 'lodash/kebabCase'
```